pkg/utils/k8sutil.go

```diff
@@ -755,3 +757,79 @@ func addInitContainerAnnotation(dpm *v1beta1.Deployment) error {
 	}
 	return nil
 }
+
+// CreateIngress creates ingress rule for a specific function
+func CreateIngress(client kubernetes.Interface, ingressName, function, domain, ns string) error {
+	if domain == "" {
+		var err error
+		domain, err = getLocalDomain()
+		if err != nil {
+			return err
+		}
+	}
+
+	//TODO: skip annotation. We can add it later
+	//ingressAnnotations := map[string]string{
```

Comment:
without this, all the lego stuff is a bit pointless.  Remove all of that too, or uncomment here?